### PR TITLE
Add manual section for requirements

### PR DESCRIPTION
**Pull request by franzTobiasDLR**
Port old requirements manual to core manual and update it to latest state of the requirements concept.

Closing #852

### CODE REVIEW

**Review by PhilMFischer — APPROVED** [review]

File: de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc [hunk 1]
```diff
@@ -1134,6 +1134,243 @@ TIP: You can also use *drag & drop* again to set the inheritance links.
...
+|Image
+
+|{counter:step}.
+|Create a RequirementsConfigurationCollection element directly in the Virtual Satellite Repository. 
+
+* This collection is a place where all requirements configuration elements should be stored
+
+TIP: User rights can be customized to prevent other users from editing this collection 
+
+Add a RequirementsConfiguration under the collection
```
[the remark this answers is not in the document]
Comment (PhilMFischer) [reply de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc]: ALready fixed it

File: de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc [hunk 2]
```diff
@@ -1134,6 +1134,243 @@ TIP: You can also use *drag & drop* again to set the inheritance links.
 
 You finalized the tutorial for the extended product structures. 
 
+==== Requirements concept
+
+To support model based requirements management and to verify requirements by using the system model, Virtual Satellite provides a concept to model requirements. Requirements can be either imported from other tools or created directly within Virtual Satellite.
```
[the remark this answers is not in the document]
Comment (franzTobiasDLR) [reply de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc]: There is now semantic-less tracing of requirements to the system model, only tracing as aprt of verification. For that we have a section in the chapter.

**Comment by SaMuellerDLR** [comment]
@franzTobiasDLR @franzTobiasDLR Its showing me the entire document as a diff, whats going on here? :D

**Comment by franzTobiasDLR** [comment]
Guess there is some line ending stuff going on... 😒

**Comment by franzTobiasDLR** [comment]
@SaMuellerDLR Fixed.

File: de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc [hunk 3]
```diff
@@ -1134,6 +1134,251 @@ TIP: You can also use *drag & drop* again to set the inheritance links.
...
+|{counter:step}.
+|To create requirements within a group click on the button `Drill-Down` to go to the group.
+|image:chapterModeling/subsectionRequirements/DrillDownReqGroup.png[Drill-Down to requirement group]
+
+|===
+
+
+===== Import / export of requirements 
+
+To integrate requirements management into the model-based systems engineering process, Virtual Satellite supports import of requirements from external tools.
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc:1237]:
```suggestion
To integrate requirements management into the model-based systems engineering process, Virtual Satellite supports importing requirements from external tools.

```

File: de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc [hunk 4]
```diff
@@ -1134,6 +1134,251 @@ TIP: You can also use *drag & drop* again to set the inheritance links.
...
+|Double-click the specification element to open its editor. 
+
+Add requirements by using the button `Add Requirement` in the requirements section.
+|image:chapterModeling/subsectionRequirements/AddRequirement.png[Add requirements]
+
+|{counter:step}.
+|When a new requirement is added, the editor asks of which type it needs to be: 
+
+* You can now choose between the types you create in the configuration element of your project. 
+The left part of the dialog allows to filter by the container element of the configuration, the right side show all applicable types in the currently selected tree element.
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc:1216]:
```suggestion
The left part of the dialog allows to filter by the container element of the configuration, the right side shows all applicable types in the currently selected tree element.

```

File: de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc [hunk 5]
```diff
@@ -1134,6 +1134,251 @@ TIP: You can also use *drag & drop* again to set the inheritance links.
...
+|image:chapterModeling/subsectionRequirements/OpenRequirementEditor.png[Open requirement in editor]
+
+|{counter:step}.
+|Add verification methods in the dedicated editor sections of the requirement.
+One requirement can also have multiple verification methods. 
+
+NOTE: Basic verification types are created in the _VerificationConfiguration_.
+
+NOTE: System model-based verification methods can be added via dedicated editor sections.
+Depending on the used concepts in the project there might be different of these model-based verification methods.
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_User_Manual.adoc:1370]:
```suggestion
Depending on the used concepts in the project, there might be different model-based verification methods. 

```

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Great work. Comments are just some minor editorial improvements.

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Pull request merged**